Ginger/GingerCoreCommon/Repository/NewRepositorySerializer.cs

```diff
@@ -671,7 +676,7 @@ private static object xmlReadObject(Object Parent, XmlReader xdr, RepositoryItem
                         }
                         else
                         {
-                            throw new Exception("NewRepositorySerializer: Unable to create class object - " + className);
+                            throw new Exception(string.Format("NewRepositorySerializer: Unable to create object for the class type '{0}'", className));
```

Comment:
Codacy found an issue: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5848147)